Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -326,6 +461,7 @@ private extension NotificationsCenterCell {
 
         // Colors
 
+        foregroundContentContainer.backgroundColor = theme.colors.paperBackground
```

Comment:
This might be caused by this line - it looks like the cell selection state background color is lost in this branch (screenshot taken from latest 6.9.0 Testflight build). Can you restore it if it's easy?

![IMG_0052](https://user-images.githubusercontent.com/3620196/146262766-d1b9a66c-5de6-4af6-987d-0d797ecc9396.PNG)

Reply:
Yeah unfortunately this work tramples over the selection state. I believe it may require a little more than changing the color there because of the stacked foreground/background containers. Let's address this separately once some of the dust in this view settles. I created a task for it: https://phabricator.wikimedia.org/T297977